cg/services/orders/submitters/order_submitter_registry.py

```diff
@@ -128,6 +131,12 @@ def get_order_submitter(self, order_type: OrderType) -> OrderSubmitter:
         StoreCaseOrderService,
         CaseOrderSubmitter,
     ),
+    OrderType.MICROBIAL_FASTQ: (
+        OrderLimsService,
+        ValidateMicrobialOrderService,
```

Comment:
Same validation as the others or do we need a different one?

Reply:
wll keep it as is now so it will get the normal sample validation treatment of microbial orders